lib/theme.json

```diff
@@ -4,6 +4,10 @@
 	"settings": {
 		"appearanceTools": false,
 		"useRootPaddingAwareAlignments": false,
+		"background": {
+			"backgroundImage": true,
+			"backgroundSize": true
+		},
```

Comment:
This turns the feature on by default.

Do folks think it's okay? We could wait until the feature is better tested and stable. 

I just added this change to discuss.

Reply:
My understanding was that background image controls should be guarded behind the `appearanceTools` flag, similar to most of the other visual controls (border, margin, padding, etc). That said, I can see how this might feel a bit more like a feature than a simple styling control, and would be useful to expose more widely. If we're thinking of changing it, I'd recommend exploring that in a separate PR as I've seen the discussions between default to on or off run on quite long for some features (most recently shadow support).

Reply:
No worries. Good points - exactly what I was after 👍🏻 

I'll remove. Many new themes have "appearanceTools" turned on anyway.